Tests/test_annotations.py

```
@@ -147,6 +147,37 @@ def test_AEnum(self):
+        def func(x: int): pass
+
+        result = Annotation.check_kwargs(func, {'a': 1, 'b': 2})
+
+        assert result == {}
+
+
+    def test_AString_empty_string_is_valid(self):
+        # Пустая строка — допустимое значение для AString, хотя "" == falsy в Python
+        input_id = AString.build(parent=self.parent)
+
+        assert AString.set(input_id, "") == True
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `AString.set(input_id, ""):` for truth checks

Replace with `AString.set(input_id, "")`

(E712)